.github/workflows/govulncheck.yml

```diff
@@ -1,4 +1,4 @@
-name: cron
+name: govulncheck
```

Comment:
nit: go-vuln-check

Reply:
`govulncheck`  is the name of the binary being used so I am fine with keeping that name

Reply:
Okay!